pkg/frontend/querymiddleware/roundtrip.go

```diff
@@ -75,9 +72,11 @@ func (cfg *Config) Validate() error {
 		if err := cfg.ResultsCacheConfig.Validate(); err != nil {
```

Comment:
I would remove it from here and just keep the one below (see the other comment).